Dockerfile

```diff
@@ -25,11 +25,41 @@ RUN apt-get update && \
   rm /unifi-video.patch && \
   chmod 755 /run.sh
 
-# Volumes
-VOLUME /var/lib/unifi-video /var/log/unifi-video
+# Base volume, will store db and configuration
+VOLUME ["/var/lib/unifi-video"]
 
-# Ports
-EXPOSE 7442 7443 7445 7446 7447 7080 6666
+# Video storage, for seperation of data
+VOLUME ["/usr/lib/unifi-video/data/videos"]
+
+# Inbound Camera Streams (NVR Side)
+EXPOSE 6666/tcp
+
+# UVC-Micro Talkback (Camera Side)
+EXPOSE 7004/udp
+
+# HTTP Web UI & API
+EXPOSE 7080/tcp
+
+# Camera Management (NVR Side)
+EXPOSE 7442/tcp
+
+# HTTPS Web UI & API
+EXPOSE 7443/tcp
+
+# Video over HTTP
+EXPOSE 7445/tcp
+
+# Video over HTTPS
+EXPOSE 7446/tcp
+
+# RTSP via the controller
+EXPOSE 7447/tcp
```

Comment:
Streaming not using UDP, isn't that crazy? But unifi's document for ports says TCP.